src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/DogFacts.java

```diff
@@ -0,0 +1,73 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks.basic;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHack;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.autoload.AutoLoad;
+import java.util.List;
+import java.util.UUID;
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.chat.TextComponent;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import vg.civcraft.mc.civmodcore.playersettings.PlayerSettingAPI;
+import vg.civcraft.mc.civmodcore.playersettings.gui.MenuSection;
+import vg.civcraft.mc.civmodcore.playersettings.impl.BooleanSetting;
+
+public class DogFacts extends BasicHack {
+
+	@AutoLoad
+	private List<String> announcements;
+	@AutoLoad
+	private long interval;
+
+	private BooleanSetting disableAnnouncements;
+
+	private int counter = 0;
+
+	public DogFacts(SimpleAdminHacks plugin, BasicHackConfig config) {
+		super(plugin, config);
+	}
+
+	@Override
+	public void onEnable() {
+		super.onEnable();
+		startRunnable(announcements, interval);
+		initSettings();
+	}
+
+	public void startRunnable(List<String> announcements, long interval){
+		Bukkit.getScheduler().scheduleSyncRepeatingTask(plugin ,() -> {
+			if (announcements.isEmpty()) {
```

Comment:
Just dont announce anything if there is nothing in the list? Also if you want to do input validation, don't do it in the runnable every time it runs